### DISCUSSION TITLE
Top bar is confusing with only icons and no text

### DISCUSSION THREAD

**Discussion by tripleo1** [community/community#69816] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

![Screenshot from 2023-10-11 12-24-58](https://github.com/community/community/assets/50311859/8abbc6c1-f5c6-4bb8-9b53-fb9c21464a2c)

**Issue**

In the top bar, I find it easier to navigate with text instead  icons as presented above (current as of 2023-Oct-11.

**Suggested Action**

Provide an option to present text based menus for users that request it.

**Comment by ayangabryl** [comment]
Okay
